src/assignments/benson/lesson_2_answers.clj

```diff
@@ -56,4 +68,37 @@
        ;; and puts the entire sequence into a set
        (vals)
        (filter #(>= (count %) 2))
-       (into #{})))
\ No newline at end of file
+       (into #{})))
+
+
+;; Returns all consecutive subsequences as nested vectors
+(defn get-consecutive-subseqs
```

Comment:
This is a very good attempt.  However you misread the spec slightly:
```
(get-consecutive-subseqs [1 4 0 1 2 3 8 3 9])
;; => [[1] [4] [0 1 2 3] [8] [3] [9]]
```

But what we want is:
```
[[1] [4] [0 1 2 3 8] [3] [9]]
```
i.e the longest subsequent of continually increasing values.  Not necessarily consecutive.